VideoApp/VideoApp/Stores/Chat/ChatFileMessage.swift

```diff
@@ -17,6 +17,7 @@
 import TwilioConversationsClient
 
 class ChatFileMessage: ChatMessage {
+    let messageType = ChatMessageType.file
```

Comment:
Look at `ChatViewModel` to see why I did this. Basically I wanted an enum so that I could rely on compiler to make sure all types were handled correctly in the UI.